src/status_im2/contexts/onboarding/profiles/view.cljs

```diff
@@ -197,18 +200,18 @@
       [quo/input
        {:type              :password
         :blur?             true
-        :override-theme    :dark
+        :disabled?         processing
         :placeholder       (i18n/label :t/type-your-password)
         :auto-focus        true
-        :error?            (when (not-empty error) error)
+        :error?            (seq error)
         :label             (i18n/label :t/profile-password)
-        :secure-text-entry true
         :on-change-text    (fn [password]
-                             (rf/dispatch [:set-in [:multiaccounts/login :password]
-                                           (security/mask-data password)])
+                             (rf/dispatch-sync [:set-in [:multiaccounts/login :password]
```

Comment:
could you elaborate?

Reply:
Yep, I mentioned you in an upper comment where I explained that.